arangod/Indexes/Index.h

```diff
@@ -481,6 +485,48 @@ struct AttributeAccessParts {
   aql::AstNodeType opType;
 };
 
+struct VectorIndexRandomVector {
+  double bParam;
+  double wParam;
+  std::vector<double> v_param;
+
+  template<class Inspector>
+  friend inline auto inspect(Inspector& f, VectorIndexRandomVector& x) {
+    return f.object(x).fields(f.field("bParam", x.bParam),
+                              f.field("wParam", x.wParam),
+                              f.field("vParam", x.v_param));
+  }
+};
+
+struct VectorIndexDefinition {
+  std::size_t dimensions;
+  double min;
+  double max;
+  std::size_t Kparamater;
+  std::size_t Lparamater;
+  std::vector<VectorIndexRandomVector> randomFunctions;
+
+  template<class Inspector>
+  friend inline auto inspect(Inspector& f, VectorIndexDefinition& x) {
+    return f.object(x)
+        .fields(f.field("dimensions", x.dimensions), f.field("min", x.min),
+                f.field("max", x.max), f.field("Kparamater", x.Kparamater),
+                f.field("Lparamater", x.Lparamater),
```

Comment:
Looks like param<b><ins>e</ins></b>ter is misspelled a few times with an **a**

Reply:
Ah you are right. I will fix that